metrics/rl_reliability/README.md

```diff
@@ -0,0 +1,116 @@
+---
+title: RL Reliability
+datasets:
+-  
+tags:
+- evaluate
+- metric
+sdk: gradio
+sdk_version: 3.0.2
+app_file: app.py
+pinned: false
+---
+
+# Metric Card for RL Reliability
+
+## Metric Description
+The RL Reliability Metrics library provides a set of metrics for measuring the reliability of reinforcement learning (RL) algorithms. 
+
+## How to Use
+
+```python
+import evaluate
+import numpy as np
+
+rl_reliability = evaluate.load("rl_reliability", "online")
+results = rl_reliability.compute(
+    timesteps=[np.linspace(0, 2000000, 1000)],
+    rewards=[np.linspace(0, 100, 1000)]
+    )
+
+rl_reliability = evaluate.load("rl_reliability", "offline")
+results = rl_reliability.compute(
+    timesteps=[np.linspace(0, 2000000, 1000)],
+    rewards=[np.linspace(0, 100, 1000)]
+    )
+```
+
+
+### Inputs
+- **timesteps** *(List[int]): For each run a an list/array with its timesteps.*
+- **rewards** *(List[float]): For each run a an list/array with its rewards.*
+
+KWARGS:
+- **baseline="default"** *(Union[str, float]) Normalization used for curves. When `"default"` is passed the curves are normalized by their range in the online setting and by the median performance across runs in the offline case. When a float is passed the curves are divided by that value.*
+- **eval_points=[50000, 150000, ..., 2000000]** *(List[int]) Statistics will be computed at these points*
+- **freq_thresh=0.01** *(float) Frequency threshold for low-pass filtering.*
+- **window_size=100000** *(int) Defines a window centered at each eval point.*
+- **window_size_trimmed=99000** *(int) To handle shortened curves due to differencing*
+- **alpha=0.05** *(float)The "value at risk" (VaR) cutoff point, a float in the range [0,1].*
+
+### Output Values
+
+In `"online"` mode:
```

Comment:
I would add the details about the metrics (from [here](https://github.com/google-research/rl-reliability-metrics/blob/ffbf8bbf1fb05717c73cebf6b57918cb071bfcd5/rl_reliability_metrics/analysis/plot_utils.py#L44)
```
   'HighFreqEnergyWithinRuns': 'High Frequency across Time (DT)',
    'IqrWithinRuns': 'IQR across Time (DT)',
    'MadWithinRuns': 'MAD across Time (DT)',
    'StddevWithinRuns': 'Stddev across Time (DT)',
    'LowerCVaROnDiffs': 'Lower CVaR on Differences (SRT)',
    'UpperCVaROnDiffs': 'Upper CVaR on Differences (SRT)',
    'MaxDrawdown': 'Max Drawdown (LRT)',
    'LowerCVaROnDrawdown': 'Lower CVaR on Drawdown (LRT)',
    'UpperCVaROnDrawdown': 'Upper CVaR on Drawdown (LRT)',
    'LowerCVaROnRaw': 'Lower CVaR on Raw',
    'UpperCVaROnRaw': 'Upper CVaR on Raw',
    'IqrAcrossRuns': 'IQR across Runs (DR)',
    'MadAcrossRuns': 'MAD across Runs (DR)',
    'StddevAcrossRuns': 'Stddev across Runs (DR)',
    'LowerCVaROnAcross': 'Lower CVaR across Runs (RR)',
    'UpperCVaROnAcross': 'Upper CVaR across Runs (RR)',
    'MedianPerfDuringTraining': 'Median Performance across Runs',
```